src/pubSubClient.js

```diff
@@ -65,13 +64,13 @@ class PubSubClient {
       });
 
       this.ws.addEventListener('error', (event) => {
-        if (this.url === defaultWsUrl) {
-          logger.error('WARNING: WebSocket connections will fail to initialize. The file has not been properly configured. Please update the URL to point to your WebSocket server for communication to work.');
-        } else {
-          logger.error('Failed to initialize a WS connection...', 'initialize');
-        }
+        logger.error('Failed to initialize a WS connection...', event);
```

Comment:
Let's leave this as "info". It will only technically be an error if someone wants to connect.